### DISCUSSION TITLE
Indicate last pushed commit

### DISCUSSION THREAD

**Discussion by tomzx** [damionx7/Zettel-Notes-Documentation#84] [Feature Requests]
In the commit list it would be great if there was a way to see the last commit that has been pushed to the remote (in the case of a git repository).

**Comment by damionx7** [comment]
Hey! There is a `List Commits` option in left navigation drawer. You can click it to view all the commits in repository.

![Screenshot_2024-06-29-10-32-41-29_377fe8767025e4fa6dddb62d6df8f716](https://github.com/damionx7/Zettel-Notes-Documentation/assets/6530843/3d1822a1-50db-424e-95b9-ac068d8beff7)

**Comment by tomzx** [comment]
It lists the commits, but what I'm after is an indication of the last *pushed* commit, which may not be the latest commit.

**Reply by damionx7** [reply]
Ok. I will add this feature.

**Reply by damionx7** [reply]
I have added this feature. It will be live in version 2.7.1
